README.md

```diff
@@ -1,8 +1,25 @@
 # Kubernetes Python Client
 
 [![Build Status](https://travis-ci.org/kubernetes-incubator/client-python.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/client-python)
+[![PyPI version](https://badge.fury.io/py/kubernetes.svg)](https://badge.fury.io/py/kubernetes)
 
-Python clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
+Python client for the [kubernetes](http://kubernetes.io/) API.
+
+## Installation
+
+From source:
+
+```
+git clone https://github.com/kubernetes-incubator/client-python.git
+cd client-python
+pip install .
```

Comment:
I think the standard way is to ```python setup.py install``` it. from [ref](https://pypi.python.org/pypi):

**Get Packages**
*To use a package from this index either "pip install package" (get pip) or download, unpack and "python setup.py install" it.*